.github/ISSUE_TEMPLATE.md

```diff
@@ -19,13 +12,9 @@ Please provide any relevant information about your setup. This is important in c
 * OpenMined Unity App Version:
```

Comment:
This whole `Test Configuration` section can probably be replaced simply with operating system type/version (windows, linux distro, mac os) and PyTorch and/or TensorFlow versions.